java/spark/src/main/java/com/logicalclocks/hsfs/spark/engine/SparkEngine.java

```diff
@@ -1097,5 +1097,14 @@ protected String makeQueryName(String queryName, FeatureGroupBase featureGroup)
     }
     return queryName;
   }
-
+  
+  public void closeSparkSession() {
+    if (getSparkSession() != null) {
+      try {
+        getSparkSession().stop();
+      } catch (Exception e) {
+        // No-OP
+      }
```

Comment:
Why catching the exception in the first place? If something happens in the `.stop()` it would be useful if it shows up in the logs, so I would say we should avoid swallowing the exception.